components/ArticleCommentTextarea/ArticleCommentTextarea.tsx

```diff
@@ -0,0 +1,54 @@
+import { ChangeEvent } from 'react';
+
+import TextareaField from '@/components/InputField/TextareaField';
+import Buttons from '@/components/Buttons';
+import { useDeviceType } from '@/contexts/DeviceTypeContext';
+
+interface ArticleCommentTextareaProps {
+  commentValue: string;
+  handleCommentChange: (e: ChangeEvent<HTMLTextAreaElement>) => void;
+  handleCommentSubmit: () => void;
+}
+
+/**
+ * @param {string} props.commentValue - textarea 입력 값
+ * @param {Function} props.handleCommentChange - textarea 입력 값 변경 이벤트
+ * @param {Function} props.handleCommentSubmit - textarea 입력 값 등록
+ * @returns {JSX.Element} - 자유게시판 게시글 상세 댓글 입력 컴포넌트
+ */
+function ArticleCommentTextarea({
+  commentValue,
+  handleCommentChange,
```

Comment:
지금은 아무것도 입력안해도 button이 disabled 상태가 안되는것같아요 이렇게 수정해주셔야 할 것 같습니다!
```suggestion
  commentValue = '',
```

Reply:
스토리북에서는 그러네요... 확인하고 수정했습니다!!